crates/sui-framework/sources/versioned_type.move

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::versioned_type {
+    use sui::object::{UID, ID};
+    use sui::tx_context::TxContext;
+    use sui::object;
+    use sui::dynamic_field;
+
+    const EInvalidUpgrade: u64 = 0;
```

Comment:
If you add a short doc (`///`) comment, here, this can get picked up for symbolication purposes (in some cases).